src/dialects/mssql/index.js

```diff
@@ -32,7 +32,7 @@ function Client_MSSQL(config = {}) {
     min: 1,
     max: 1,
     idleTimeoutMillis: Number.MAX_SAFE_INTEGER,
-    evictionRunIntervalMillis: 0,
+    reapIntervalMillis: 1000,
```

Comment:
Actually right now this can be removed completely since default value is also 1000 and idleTimeTimeout is so big that nothing will be released ever because of idling anyways.

Reply:
that way pool parameters should be compatible with generic-pool and tarnjs